scripts/operation.py

```diff
@@ -104,38 +99,50 @@ def get_signer() -> Any:
         help='Transactions file path. Each transaction per line',
         required=True,
     )
+    current_parser.add_argument(
+        "--phase",
+        type=str,
+        help='Phase of deploy: preparation, deactivation, upgrade, adding',
+        required=True,
+    )
 
     current_parser = subparsers.add_parser(
-        'check-transactions',
-        help='Check transactions from a file',
+        'check-transactions', help='Check transactions from a file'
     )
     current_parser.add_argument(
-        "--transactions-path",
+        "--phase",
         type=str,
-        help='Path to transactions file',
+        help='Phase of deploy: preparation, deactivation, upgrade, adding',
         required=True,
     )
+    current_parser.add_argument(
+        "--transactions-path", type=str, help='Path to transactions file', required=True
+    )
+
+    current_parser = subparsers.add_parser('test', help='`Command for tests`')
 
     args = parser.parse_args()
 
     sys.argv.append('--verbose')
 
-    with open(args.config) as f:
+    install_solido()
+    with open(os.getenv("SOLIDO_CONFIG")) as f:
```

Comment:
This is not a good idea. We should not use hard coded environment variables. You could just pass it as an argument 
```python
lido_state = solido('--config', os.getenv("SOLIDO_CONFIG"), 'show-solido')
```